tl/controllers/middleware.rst

```diff
@@ -280,32 +279,32 @@ application::
     {
         public function middleware($middlewareQueue)
         {
-            // Add your simple middleware onto the queue
+            // Idagadg sa iyong simple na middleware papunta sa queue
```

Comment:
- Idagdag
- should we translate queue or not?

Reply:
i think lets leave queue untranslated.

Reply:
done change